GDBrowser/dist/presence.js

```diff
@@ -0,0 +1,167 @@
+var presence = new Presence({
+    clientId: "635876670146084880"
+})
+
+presence.on("UpdateData", async () => {
+    let presenceData = {
+        largeImageKey: "coin"
+    }
+
+    const q = new URLSearchParams(window.location.search);
+
+    if (window.location.href.includes("gdbrowser.com")) {
+
+            // LevelData is just for regular levels
+            function LevelData(){
+                let downloads = document.getElementsByClassName("inline smaller spaced")[0].innerHTML
+                let likes = document.getElementsByClassName("inline smaller spaced")[1].innerHTML
+                let orbs = document.getElementsByClassName("orbs")[1].innerHTML
+                presenceData.state = `🔽 ${downloads} | 👍 ${likes} | 🔵 ${orbs}`;
+                presenceData.details = `${document.getElementsByTagName("h1")[0].innerText} ${document.getElementById("authorLink").innerText}`
+                presenceData.smallImageKey = `diff_${document.getElementById("difficultytext").innerHTML.toLowerCase().replace("<br>", "_")}`
+                presenceData.smallImageText = `${document.getElementById("difficultytext").innerHTML.replace("<br>", " ")}`
+            }
+
+            // If the url has numbers at the end (/)
+            if (!isNaN(window.location.pathname.toLowerCase().replace("/", "")) && window.location.pathname.toLowerCase() !== "/"){
+                LevelData()
+            }
+
+            if (window.location.pathname.toLowerCase() === "/daily") {
+                LevelData()
+            }
+            
+            if (window.location.pathname.toLowerCase() === "/weekly") {
```

Comment:
You can also use or (`||`) operator to check for multiple things in a single if statement. If you're going to use the same function for all these different pages, you can just do `if (one || two && two2 || three || goesOn) { }`

Reply:
Ahh, ok. Gotcha

Reply:
Sorry for the late addition, I was unable to check the code more carefully 😓